packages/manager/modules/web-paas/src/details/service/service.controller.js

```diff
@@ -0,0 +1,10 @@
+export default class WebPassDetailsServiceCtrl {
+  /* @ngInject */
+  constructor(Alerter) {
+    this.Alerter = Alerter;
+  }
+
+  onBillingInformationError(error) {
+    this.Alerter.error(error.message, 'web_paas_dashboard_alert');
```

Comment:
no need for translation? below is an example of using it
this.alerter.error(
          `${this.$translate.instant('user_users_error')} ${get(
            err,
            'message',
            err,
          )}`,
          'userUsers',
        );